extension/js/common/platform/catch.ts

```diff
@@ -281,6 +281,14 @@ export class Catch {
     }
   }
 
+  private static groupSimilarReports(value: string): string {
+    return value
+      .replace(/chrome-extension:\/\/[^\/]+\//, 'chrome-extension://EXTENSION_ID/')
+      .replace(/https:\/\/www\.googleapis\.com\/gmail\/v1\/users\/me\/threads\/[^\/]+\//, 'https://www.googleapis.com/gmail/v1/users/me/threads/THREAD_ID/')
+      .replace(/https:\/\/www\.googleapis\.com\/gmail\/v1\/users\/me\/messages\/[^\/]+\//, 'https://www.googleapis.com/gmail/v1/users/me/messages/MESSAGE_ID/')
+      .replace(/https:\/\/www\.googleapis\.com\/gmail\/v1\/users\/me\/drafts\/[^\/]+\//, 'https://www.googleapis.com/gmail/v1/users/me/drafts/DRAFT_ID/');
+  }
```

Comment:
currently only `EXTENSION_ID` replacement works, in other regexes trailing slashes should be removed - as urls look like https://www.googleapis.com/gmail/v1/users/me/messages/12079801d03027dc

Reply:
Ohhh. misssed it.
Just fixed it